Makefile

```diff
@@ -1,4 +1,39 @@
 # There's no heavy lifting here; this file is just a convenience
```

Comment:
Note the changes below make this comment sort of a lie now :smile: 

Maybe say: "Avoid adding substantial dedicated code here, more than 5 lines of inline shell script for a rule should invoke an external tool"?

Reply:
... I'll just remove the comment :-D